config.go

```diff
@@ -203,10 +203,11 @@ type ToznySDKJSONConfig struct {
 	// Embed all config for v1 and v2 clients
 	ConfigFile
 	TozIDSessionIdentityData `json:"toz_id_session_identity_data"`
-	PublicSigningKey  string `json:"public_signing_key"`
-	PrivateSigningKey string `json:"private_signing_key"`
-	AccountUsername   string `json:"account_user_name"`
-	AccountPassword   string `json:"account_password"`
+	PublicSigningKey         string  `json:"public_signing_key"`
+	PrivateSigningKey        string  `json:"private_signing_key"`
+	AccountUsername          string  `json:"account_user_name"`
+	AccountPassword          string  `json:"account_password"`
+	AgentToken               *string // the tozid-realm-idp client jwt
```

Comment:
comment please on why we're intentionally not persisting this to a saved config file (assuming thats the right call)